packages/core/scripts/liquidity-mining/JoinRolledPayouts.js

```diff
@@ -23,7 +23,7 @@ async function JoinRolledPayouts(week, rollNum, tokenName) {
     throw new Error("Missing or invalid parameter! Provide week, rollNum & tokenName");
```

Comment:
small logic change in the joiner to consider the different ways that the script can be used